django_project/frontend/src/components/Table/ServerTable.tsx

```diff
@@ -121,16 +136,19 @@ const ServerTable = forwardRef(
         page: 0,
         page_size: pageSize,
         sort: getSort(defaults.sort),
-        ...defaults.filters
+        ...defaults.filters,
+        ...filterModel
       }
     )
 
     useEffect(() => {
+      window.sessionStorage.setItem(url, JSON.stringify(cleanFilters(filterModel)))
```

Comment:
We don't need this
Just everytime the parameter is changed, change the url after hash